arm/ripper/handbrake.py

```diff
@@ -6,6 +6,9 @@
 import subprocess
 import re
 import shlex
+import time
+import datetime
+import psutil
```

Comment:
I'm not familiar with Python enough to know if this import is necessary, but it doesn't seem to be used in the changes you've added. Is this because `utils.SleepCheckProcess` on line 32 depends on `psutil`?

Reply:
Remanent from when I had the function built in. Will remove in next release.